src/libraries/Pool.sol

```diff
@@ -191,7 +191,7 @@ library Pool {
                 (uint256 feesOwed0, uint256 feesOwed1) =
                     position.update(liquidityDelta, feeGrowthInside0X128, feeGrowthInside1X128);
 
-                // Fees earned from LPing are added to the user's currency delta.
+                // Fees earned from LPing are calculated, and returned
```

Comment:
Can we still comment somewhere that fees will be added to the final delta or when we do the addition, comment why fees get added

Reply:
adding a comment where they get added together